authorization/client/ownerCache.go

```diff
@@ -109,6 +109,9 @@ func (d *ownerSubject) syncDevicesLocked(ctx context.Context, owner string, cach
 	return added, removed, nil
 }
 
+// ErrFunc reports errors
+type ErrFunc func(err error)
```

Comment:
change `type ErrFunc func(err error)` to alias `type ErrFunc = func(err error)` we don't need to be strict